pkg/cmd/secret/remove/remove_test.go

```diff
@@ -155,36 +200,46 @@ func Test_removeRun_env(t *testing.T) {
 
 func Test_removeRun_org(t *testing.T) {
 	tests := []struct {
-		name string
-		opts *RemoveOptions
+		name     string
+		opts     *RemoveOptions
+		wantPath string
 	}{
 		{
-			name: "repo",
-			opts: &RemoveOptions{},
+			name:     "repo",
```

Comment:
It seems like this test and the "Dependabot repo" test are duplicates of the tests above in `Test_removeRun_repo`? Not sure both are necessary.

Reply:
These were duplicated by mistake, removed in https://github.com/cli/cli/pull/5160/commits/5eb19f9bf36f7d7985009de4b4d88b9bdc3c274a